guides/demo.md

```diff
@@ -0,0 +1,168 @@
+
+### YAML Structure
+
+The Configuration is saved in [fake_facts.yaml](/priv/demo/fake_facts.yaml) file, which follows a specific structure with two main sections: targets and facts.
+
+Example:
+
+```yaml
+targets:
+  target1: 0a055c90-4cb6-54ce-ac9c-ae3fedaf40d4
+  target2: 13e8c25c-3180-5a9a-95c8-51ec38e50cfc
+facts:
+  check_1:
+    fact_name1:
+      target1: 2
+      target2: 3
+```
+
+#### Targets
+
+The targets section allows defining handles for targets' UUIDs, providing a reference that can be used through the configuration.
+
+Format:
+
+```yaml
+targets:
+  <target_reference>: <target_id>
+```
+
+<target_reference>: A user-defined handle or reference for a target.
+
+<target_id>: The UUID or identifier of the target.
+
+#### Add a new target to the demo configuration
+
+Open the [fake_facts](/priv/demo/fake_facts.yaml) configuration and add a new target entry to the targets section, following this format:
+
+```
+targetX: <new_target_id>
+```
+
+Replace targetX with an unique target identifier (e.g target3) and <new_target_id>
+
+Add the new target entry:
+```yaml
+targets:
+  target1: 0a055c90-4cb6-54ce-ac9c-ae3fedaf40d4
+  target2: 13e8c25c-3180-5a9a-95c8-51ec38e50cfc
+  target3: <new_target_id>
+```
+
+### Facts Section:
+
+The facts section allows specifying the facts associated with specific targets and checks. Each fact corresponds to a piece of information that a target can return during the evaluation of a check.
+
+The format for the facts section is as follows:
+
+```yaml
+facts:
+  <check_id>:
+    <fact_name>:
+      <target_reference>: <fact value>
+      <target_reference2>: <fact value>
+```
+
+#### Add a new Fact to the demo configuration
+
+Open the [fake_facts](/priv/demo/fake_facts.yaml) configuration and add a new fact to the facts section, following this format:
+
+```yaml
+facts:
+  <check_id>:
+    <fact_name>:
+      target1: <fact value>
+      target2: <fact value>
+      target3: <fact value>
+```
+
+Example:
+
+```yaml
+targets:
+  target1: 0a055c90-4cb6-54ce-ac9c-ae3fedaf40d4
+  target2: 13e8c25c-3180-5a9a-95c8-51ec38e50cfc
+  target3: <new_target_id>
+
+facts:
+.......existing facts..........
+  new_check_id:
+    new_fact_name:
+      target1: <fact value>
+      target2: <fact value>
+      target3: <fact value>
+```
+
+## FAQ
+
+### How to apply the configuration changes?
+
+Save the [fake_facts](/priv/demo/fake_facts.yaml) configuration file and re-run executions from the [web](https://github.com/trento-project/web) ui
+
+### What happens with unmapped targets, facts or checks?
+
+Wanda will still evaluate and return a default fallback fact value "some fact value" for a check's execution. This means when a new check is added to wanda, changing the the configuration is optional.
+
+### How to know what's the correct return value of a fact?
```

Comment:
We can shrink this as well by providing just a reference to the gatherers and the catalog.

See [link to gatherers] and [link to catalog] for extra information about what checks expect.

Reply:
Done!